dotnet-authserver/src/TeacherIdentity.AuthServer/Services/UserImport/ServiceCollectionExtensions.cs

```diff
@@ -29,11 +29,12 @@ public static class ServiceCollectionExtensions
             });
 
             services.AddSingleton<IUserImportStorageService, BlobStorageUserImportStorageService>();
-            services.AddSingleton<INameSynonymsService, NameSynonymsService>();
-            services.AddScoped<IUserSearchService, UserSearchService>();
             services.AddScoped<IUserImportProcessor, UserImportProcessor>();
         }
 
+        services.AddSingleton<INameSynonymsService, NameSynonymsService>();
+        services.AddScoped<IUserSearchService, UserSearchService>();
+
```

Comment:
Brought these out so they could be found in tests. Can potentially do the same for UserImportProcessor